docs/ossm/integrations/spire/resources/bookinfo.yaml

```diff
@@ -0,0 +1,346 @@
+# Copyright Istio Authors
```

Comment:
I think it's better to reference the upstream YAML instead of storing it here.
If could be referenced as `master` - https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/platform/kube/bookinfo.yaml
Or any other specific release.